### PR TITLE
Fixed an issue where psql was not taking the role used to connect in server properties. #5807

### PR DESCRIPTION
**Pull request by pravesh-sharma**

### CODE REVIEW

File: web/pgadmin/tools/psql/__init__.py [hunk 1]
```diff
@@ -531,50 +561,22 @@ def get_connection_status(conn):
...
         is_connected = get_connection_status(conn)
 
-        if is_connected:
-
-            if conn.manager and conn.manager.db_info \
-                    and conn.manager.db_info[int(did)] is not None:
-
-                db_name = conn.manager.db_info[int(did)]['datname']
-                return db_name
-            elif sid:
```
Comment (nikhil-mohite) [inline web/pgadmin/tools/psql/__init__.py:553]: Any specific reason to remove the else?

**Review by nikhil-mohite — CHANGES_REQUESTED** [review]

File: web/pgadmin/tools/psql/__init__.py (hunk 1 above)
Comment (pravesh-sharma) [reply web/pgadmin/tools/psql/__init__.py:553]: I discussed this with Khusboo and she suggested else part would never be executed so removed.

**Comment by nikhil-mohite** [comment]
Hi @pravesh-sharma,

Issues found in current PR
1. If we have a role that has special characters in it, the set role fails in the PSQL tool.
image
image

File: web/pgadmin/tools/psql/__init__.py (hunk 1 above)
Comment (khushboovashi) [reply web/pgadmin/tools/psql/__init__.py:553]: The code written was not necessary. We can get the database name from the connection itself; there is no need for a server manager.

**Review by nikhil-mohite — APPROVED** [review]

**Pull request merged**